openpype/hosts/nuke/api/lib.py

```diff
@@ -2118,6 +2128,42 @@ def set_root_colorspace(self, imageio_host):
             if correct_settings:
                 self._set_ocio_config_path_to_workfile(config_data)
 
+    def _get_monitor_settings(self, viewer_lut, monitor_lut):
+        """ Get monitor settings from viewer and monitor lut
+
+        Args:
+            viewer_lut (str): viewer lut string
+            monitor_lut (str): monitor lut string
+
+        Returns:
+            dict: monitor settings
+        """
+        output_data = {}
+        m_display, m_viewer = get_viewer_config_from_string(monitor_lut)
+        v_display, v_viewer = get_viewer_config_from_string(
+            viewer_lut
+        )
```

Comment:
```suggestion
        m_display, m_viewer = get_viewer_config_from_string(monitor_lut)
        v_display, v_viewer = get_viewer_config_from_string(viewer_lut)
```
Not sure why the second line was on two lines even though it's a shorter line. :)